feat: headers components

Anderson22-dev opened the pull request:
Criação dos componentes dos headers

Anderson22-dev committed:
feat: headers components

Anderson22-dev committed:
Co-authored-by: Lucas Bezerra <bezerra-lucas@users.noreply.github.com>

Anderson22-dev committed:
Feature: Header final

Eduardo-BarrosoS reviewed (changes requested):
You should create accordion in another branch

Anderson22-dev committed:
Fix: deleted accordion component

Anderson22-dev committed:
Fix: deleted accordion components

qalexandre commented:
@Anderson22-dev  faça um pull da develop primeiro, resolva os conflitos e sobe as alterações

Anderson22-dev committed:
Merge branch 'develop' of https://github.com/devcodapp/design-system into feature/headers

Eduardo-BarrosoS commented on package.json:
```diff
@@ -1,28 +1,34 @@
+        "start": "next start",
+        "lint": "next lint"
+    },
+    "dependencies": {
```
Replace fontawesome to lucide react lib

Eduardo-BarrosoS commented on src/app/page.tsx:
```diff
@@ -1,13 +1,16 @@
 import { Text } from '@/components/Text'
 
```
If you are not using Text component, you can delete this one

Eduardo-BarrosoS commented on src/app/page.tsx:
```diff
@@ -1,13 +1,16 @@
 
+import * as React from "react";
+import Header from "@/components/Header";
 
```
If you are not using React import, you can delete this one

Eduardo-BarrosoS commented on src/components/Header.tsx:
```diff
@@ -0,0 +1,31 @@
+            {secondaryActionBtn? secondaryActionBtn: null}
+        </div>
+    </div>
+    <div className="flex flex-row items-center">
```
![image](https://github.com/devcodapp/design-system/assets/75212547/79c60abd-7015-4e00-b97c-a1318bbefc00)

You need to add a space between buttons

Anderson22-dev committed:
fix: resolve github comments.

Eduardo-BarrosoS commented on tailwind.config.ts:
```diff
@@ -121,6 +132,8 @@ const config: Config = {
-  plugins: [],
+  darkMode: "class",
+  
+  plugins: [nextui()],
```
Are you sure you need this plugin and why?

Eduardo-BarrosoS commented on src/components/Header.tsx:
```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from "react";
+
```
You have to develop pixel perfect what is on figma design.
It means that this component should be responsive.

Anderson22-dev committed:
fix: responsibility header

Eduardo-BarrosoS committed:
feat: tailwind config error

Eduardo-BarrosoS committed:
feat: develop pixel perfect based on figma layout

Eduardo-BarrosoS committed:
Merge branch 'feature/headers' of https://github.com/Eduardo-BarrosoS/design-system into feature/headers

Eduardo-BarrosoS committed:
Remove redundant page title from HomePage
component

Removed an obsolete static title from the HomePage, simplifying the
layout and relying on dynamic header components for content structure.
This change aligns with the design system's directive to ensure all page
titles are managed centrally by header components, improving consistency
across the application.

Eduardo-BarrosoS committed:
Merge branch 'develop' into feature/headers

Eduardo-BarrosoS commented:
> You should create accordion in another branch

Eduardo-BarrosoS reviewed (approved):
Approved
